googleapis/google-cloud-node
# migrate code from googleapis/nodejs-orchestration-airflow

- feat: initial files
- feat!: initial release of orchestration-airflow (#2)
- chore: release 1.0.0 (#4)
- chore(deps): update dependency sinon to v11 (#6)
- chore(deps): update dependency mocha to v9 (#5)
- build(node): update deps used during postprocessing (#1243) (#7)
- docs: fix docstring formatting (#8)
- Revert "build(node): update deps used during postprocessing (#1243) (#7)" (#11)
- docs: fix docstring formatting (#9)
- chore(deps): update dependency @types/node to v16 (#10)
- chore: fix the wrong post processor image (#12)
- chore(deps): update dependency sinon to v12 (#13)
- docs(samples): add example tags to generated samples (#14)
- build: add generated samples to .eslintignore (#15)
- docs(node): support "stable"/"preview" release level (#1312) (#18)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#19)
- chore: add api_shortname and library_type to repo metadata (#17)
- test(nodejs): remove 15 add 16 (#1322) (#21)
- chore: update v2.12.0 gapic-generator-typescript (#25)
- chore(deps): update dependency sinon to v13 (#27)
- docs(nodejs): version support policy edits (#1346) (#29)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#30)
- chore: update v2.14.2 gapic-generator-typescript (#34)
- build(node): update client library version in samples metadata (#1356) (#41)
- chore(deps): update dependency sinon to v14 (#44)
- build!: update library to use Node 12 (#46)
- chore(deps): update dependency jsdoc-region-tag to v2 (#50)
- chore(deps): update dependency jsdoc-fresh to v2 (#49)
- chore(main): release 2.0.0 (#47)
- feat: support regapic LRO (#51)
- chore(main): release 2.1.0 (#52)
- chore(deps): update dependency linkinator to v4 (#56)
- chore: remove unused proto imports (#62)
- fix: better support for fallback mode (#63)
- fix: change import long to require (#64)
- fix: do not import the whole google-gax from proto JS (#1553) (#67)
- fix: allow passing gax instance to client constructor (#68)
- chore(main): release 2.1.1 (#65)
- fix: preserve default values in x-goog-request-params header (#69)
- test: use fully qualified request type name in tests (#72)
- fix(deps): use google-gax v3.5.2 (#76)
- chore(deps): update dependency @types/node to v18 (#74)
- fix: regenerated protos JS and TS definitions (#79)
- chore(deps): update dependency jsdoc to v4 (#78)
- chore(main): release 2.1.2 (#70)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
